### ISSUE TITLE
Task handler GUI/service discrepancy

### ISSUE THREAD

**Issue by PeterGhimself** [space-concordia-robotics/robotics-prototype#109]
Environment info:
- `SkidSteering2.ino` used for rover MCU

Steps to reproduce (according to @vashmata)
1. Odroid plugged into mux board on rover
2. Laptop connected to Odroid via Ethernet
3. Run `./app.py` and `roslaunch rosbridge_server rosbridge_websocket.launch`
4. Go to rover page: `localhost:5000/rover`
5. Choose the rover mux channel and click the rover listener toggle button to enable it.

Expected output:
If you're on rover mux channel, it will respond in 3-4sec telling you it opened the listener. But if you try pinging it will fail, and if you refresh the page the listener will be off.

Actual output:
?

Error messages:
?

**Comment by PeterGhimself** [comment]
Ran a local test. The only issue I ran into was not having [geometry_msgs](http://wiki.ros.org/geometry_msgs) installed/built on my machine. This would cause `RoverNode.py` to crash because dependencies were not found. The README should be updated with this, or we should add the package into our src folder. I'm honestly not sure which approach is best in this case.
I can try reproducing the bug again with the actual rover setup and see what the issue is.
